example/sinatra/index.erb

```diff
@@ -91,18 +91,21 @@
                 openDialog(redirect_url);
             },
             onSuccess: function(response){
+                document.getElementById('result-json').innerHTML += JSON.stringify(response, null, 2);
                 console.log('success');
                 console.log('response:',response);
                 closeDialog();
```

Comment:
are you sure this issue I told you earlier solved? Because I don't see `$('button').removeAttr("disabled");` here.
> can't seems to attempt another payment after success even. 

Are you copying this Core API HTML template from PHP? looks like it is a bad example i'll ask the maintainer to fix it too, you should copy from Go or NodeJS example

Reply:
Yes kak i'm copying from PHP example. Thank you for your direction